terraform/deploy/cognito/modules.tf

```diff
@@ -7,6 +7,7 @@ module "cognito" {
     "jupyterhub",
   ]
 
-  root_dns_names = values(local.root_dns_name)
-  domain         = local.cognito_domain
+  root_dns_names  = values(local.root_dns_name)
+  domain          = local.cognito_domain
+  pre_auth_lambda = data.terraform_remote_state.analytical-frontend-service.outputs.pre_auth_lambda
```

Comment:
Concourse is failing on this due to no lambdas being in front-end service currently

Reply:
You will also need to grab the "arn" of the lambda, not the object of the lambda.

Reply:
Fernando has another PR open to fix that - https://github.com/dwp/dataworks-analytical-frontend-service/pull/25

Reply:
Thanks guys.